pkgs/applications/graphics/cewe-fotobuch/install.pl

```diff
@@ -0,0 +1,973 @@
+#! /usr/bin/perl -w

```

Comment:
It's actually the installer.

Full stop: This file downloads the actual installation files without ```fetchurl``` etc. This won't work with chrooted builds. 

This needs to be fixed before we can merge this.